SSA/Projects/InstCombine/LLVM/Semantics.lean

```diff
@@ -185,7 +207,7 @@ def udiv {w : Nat} (x y : IntW w) (flag : ExactFlag := {exact := false}) : IntW
   let y' ← y
   let exact := flag.exact
   let Exact? : Prop := exact ∧
-    (x'.toNat % y'.toNat != 0)
+    (x'.umod y' ≠ 0)
   if Exact? then
```

Comment:
I guess we can clean this up as well by folding things together as suggested above.